src-tauri/src/editor/mod.rs

```diff
@@ -97,16 +100,24 @@ pub async fn export_gif(options: ExportOptions, app_handle: AppHandle) {
 
     let gif_encoder = Arc::new(gif_encoder);
 
-    let gif_name = chrono::Local::now()
+    let _gif_name = chrono::Local::now()
```

Comment:
Can you try using this as a default gif file name, user can edit it too, but if he just wants to quickly save it... A predefined name would be better

Reply:
Ok.. I've added an option to set a default filename based on the current date and time